frontend/src/components/listofusers/ListElement.js

```diff
@@ -14,7 +14,7 @@ const ListElement = ({ user }) => {
       </td>
       <td>{user.dateOfRegistration}</td>
       <td>
-        <ApproveButton user={user} />
+        <ApproveOrDeleteButton user={user} />
```

Comment:
Lehet ennek az eldöntését ide kellene inkább rakni, code smell, hogy ha valaminek a nevében benne van az OR, mert már kívülről ordít, hogy két egymástól különböző, nem összetartozó dolgot csinál. Attól, hogy ugyanott van és jelenleg egyszerre sosem látni őket még nem ugyanaz a kettő.
Például majd jön a change request, hogy lehessen törölni még nem approve-olt usert is... Akkor ApproveOrDeleteOrApproveAndDeleteButton lesz a komponens neve? :)

Reply:
B variáció: legyen OperationButtons, ami elrejti a logikát, de akkor is kellene külön-külön komponens legyen az ApproveButton meg a DeleteButton szerintem.

Reply:
Nekem is halpiac szagú, furcsa volt, de úgy tűnik nem eléggé. :)